tests/users.test.ts

```diff
@@ -100,6 +100,7 @@ describe('Users', () => {
   const confPass = '12345678'
   const oldSecret = '12345678'
   const newSecret = '87654321'
+  const hostUrl: string = 'http://localhost'
```

Comment:
Can we have a realistic `hostUrl` such as `gmail.com` or `smtp.mailtrap.io`

Reply:
This is fine for tests

Reply:
We don't have anywhere where the hostUrl example is set. That's why I asked.